build-system/tasks/extension-generator/extension-doc.template.md

```diff
@@ -64,13 +59,93 @@ limitations under the License.
   </tr>
 </table>
 
-## Behavior
+<!--
+  If the component is relevant for more than one format and operates differently between these
+  formats, include and filter multiple code samples.
```

Comment:
The comment makes it sound like filtering can only be used for code samples, however, it can be used for all content.